storage/block/block.go

```diff
@@ -169,6 +180,7 @@ func (b *dbBlock) Seal() {
 	b.writable = false
 	if stream := b.encoder.Stream(); stream != nil {
 		b.segment = stream.Segment()
+		b.checksum = digest.SegmentChecksum(b.segment)
```

Comment:
There is no `Unseal` method for the block, once it's sealed, it'll remain sealed. We do reset the checksum when the block is reset.